<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Glowna</class>
 <widget class="QMainWindow" name="Glowna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Glowna</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_bazadanych">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Baza danych</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClose_wyjdz">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>390</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Zamknij</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>291</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wybierz restaurację</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_restauracje"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Wróć</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>60</y>
        <width>391</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_restauracja">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Restauracja&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView_produkty">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <signal>rejected()</signal>
 </slots>
</ui>
